examples/hello/bpTimeWriter/helloBPTimeWriter.py

```diff
@@ -19,27 +19,34 @@
 # User data
 myArray = np.array([0, 1, 2, 3, 4, 5, 6, 7, 8, 9])
 nx = myArray.size
+time = np.array([0.0])
 
 # ADIOS
 adios = adios2.ADIOS(comm, adios2.DebugON)
 
 # IO
 bpIO = adios.DeclareIO("BPN2N")
+bpIO.SetEngine('bp3')
```

Comment:
Does this also work with the default BP4 engine?

Reply:
If you change 'bp3' to 'bp4', yes.